### DISCUSSION TITLE
When I load a world using AnvilLoader, there are a lot of Chunks that are not loaded, why is that?

### DISCUSSION THREAD

**Discussion by clok-1201** [Minestom/Minestom#2309] [Help] [answered]
![image](https://github.com/user-attachments/assets/46c3daa7-96f7-4c9b-9e8b-1dc5d3f3728e)
When I load a world using AnvilLoader, there are a lot of Chunks that are not loaded, why is that?

**Comment by wiidotmom** [comment, marked as answer]
The world was likely created in an older version and never DataFixerUpper'd. I'd bet you all the missing chunks contain short grass.

**Reply by clok-1201** [reply]
Yes！I've fixed this, thanks!
